admin/users/views.py

```diff
@@ -0,0 +1,40 @@
+from django.shortcuts import redirect
+
+from website.project.model import User
+
+from admin.base.views import GuidFormView, GuidView
+from admin.users.templatetags.user_extras import reverse_user
+from .serializers import serialize_user
+
+
+def disable_user(request, guid):
+    user = User.load(guid)
+    # user.disable_account()
+    user.is_disabled = True  # TODO: change to user.disable_account() after local tests
```

Comment:
@sloria I get a `ConnectionError: nodename nor servname provided, or not known` This isn't in the standard mailchimp error library like the other error that is caught and passed in the `disable_account` 

If I take out a fake api key '2-2' then I just get a straight-up `RuntimeError: An API key is required to connect to Mailchimp.` Not a very subtle error.